scripts/main.js

```diff
@@ -0,0 +1,198 @@
+    })
+    event.target.reset();
+  })
+
+
+
+
+// ********************** category  **********************
+
+
+document.querySelector("#category form").addEventListener("submit", (event) => {
+  event.preventDefault();
+  const category = event.target.category.value;
+  const categoryUrl = "https://emagi-server-8-0.herokuapp.com/categories/"
+
+
+  if (category === "all") {
+ fetch("https://emagi-server-8-0.herokuapp.com/emojis").then((response) => response.json()).then(displayCategory).catch(console.log)
+ if (document.querySelector("#category aside").style.background = "var(--color-red)") {
+  document.querySelector("#category aside").style.background = "var(--color-green)"
+}
+  }
+  else if (category === "default") {
+
+    document.querySelector("#category aside p").textContent = "Choose a category"
+    document.querySelector("#category aside").classList.add("error");
+    if (document.querySelector("#category aside").style.background = "var(--color-green)") {
+      document.querySelector("#category aside").style.background = "var(--color-red)"
+    }
+  }
+
+  else {
+
+  fetch(categoryUrl + category).then((response) => response.json()).then(displayCategory).catch(console.log)
+  if (document.querySelector("#category aside").style.background = "var(--color-red)") {
+    document.querySelector("#category aside").style.background = "var(--color-green)"
+  }
+ 
+  }
+
+  event.target.reset();
+})
+
+// for then
+const displayCategory = (emojis) => {
+  const getSymbol = emojis.map((emoji) => emoji.symbol)
+  document.querySelector("#category aside p").textContent = getRandom(getSymbol);
+  document.querySelector("#category aside").classList.add("success");
+  
+  return getSymbol
+  
+}
+
+
+
+// ********************** replace  **********************
+
+
+document.querySelector("#replace form").addEventListener("submit", (event) => {
+
+  event.preventDefault();
+  const input = event.target.replaced.value;
+  
+  fetch(emojiURL).then((response) => response.json()).then((emojis) => {
+
+
+  document.querySelector("#replace aside p").textContent = replace(input, emojis);
+  document.querySelector("#replace aside").classList.add("success");
+  document.querySelector("#replace aside").classList.remove("error")
+  if (document.querySelector("#replace aside").style.background = "var(--color-red)") {
+    document.querySelector("#replace aside").style.background = "var(--color-green)"
+  }
+
+  }).catch((error) => {
+     if (!input) {
+     document.querySelector("#replace aside p").textContent = "Please enter something"
+      document.querySelector("#replace aside").classList.add("error");
+      document.querySelector("#replace aside").classList.remove("success")
+      if (document.querySelector("#replace aside").style.background = "var(--color-green)") {
+        document.querySelector("#replace aside").style.background = "var(--color-red)"
+      }
+     }
+  })
+
+
+  event.target.reset();
+
+})
+const replace2 = (words, db) => {
```

Comment:
It looks like you created this second replace function but it is never actually used. Did you mean to use this one instead of the one in the replace file?